### PR TITLE
CI: fix and test build.yml

### PR DESCRIPTION
**Pull request by SMillerDev**
CI never ran so I didn't see the issue, sorry to bother you again @Bo98

### CODE REVIEW

**Comment by SMillerDev** [comment]
Not sure why that is an issue 🤷 . Maybe we just rewrite it?
```
Your Ruby version is 3.2.2, but your Gemfile specified ~> 2.6.0
```

**Comment by Bo98** [comment]
> Not sure why that is an issue 🤷 . Maybe we just rewrite it?
> 
> ```
> Your Ruby version is 3.2.2, but your Gemfile specified ~> 2.6.0
> ```

Yeah this is intentional - but looks like you've already found the env to control that.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: Build
 
 on:
+  pull_request:
+    paths:
+      - .github/workflows/build.yml
```
Comment (Bo98) [inline .github/workflows/build.yml:6]: `tests.yml` already invokes this so this should not be needed.

Comment (SMillerDev) [reply .github/workflows/build.yml:6]: Removing this line triggered nothing though

Comment (Bo98) [reply .github/workflows/build.yml:6]: It did last time: https://github.com/Homebrew/homebrew-portable-ruby/pull/173/checks

Comment (SMillerDev) [reply .github/workflows/build.yml:6]: Let's give it another go

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by SMillerDev** [comment]
> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

😢

**Comment by MikeMcQuaid** [comment]
@SMillerDev you now have it

**Pull request merged**